mx.py

```diff
@@ -10694,6 +10694,7 @@ def __call__(self, msg):
              '-doclet', 'org.apidesign.javadoc.codesnippet.Doclet',
              '-docletpath', snippetslib,
              '-snippetpath', snippets,
+             '-verifysincepresent',
```

Comment:
I guess you should also add the same change to [line 10600](https://github.com/graalvm/mx/pull/62/files#diff-1850d7da100ab46a504d0dd5236dc495R10600), right?